admin/app/controllers/authenticated/certifications/certification/details.js

```diff
@@ -0,0 +1,91 @@
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { alias } from '@ember/object/computed';
+import Controller from '@ember/controller';
+import { schedule } from '@ember/runloop';
+
+export default class CertificationDetailsController extends Controller {  // Properties
+  juryRate = false;
+
+  juryScore = false;
+  requestedId = '';
+
+  // Private properties
+  @service('mark-store')
+  _markStore;
+
+  // Aliases
+  @alias('details.percentageCorrectAnswers')
+  rate;
+
+  @alias('details.totalScore')
+  score;
+
+  @alias('model')
+  details;
+
+  @action
+  onUpdateRate() {
+    const competences = this.get('details.competences');
+    const { good, count, jury } = _getCertificationResultsAfterJuryUpdate(competences);
+
+    if (jury) {
+      this.set('juryRate', Math.round(good * 10000 / count) / 100);
+    } else {
+      this.set('juryRate', false);
+    }
+    // TODO: find a better way
+    schedule('afterRender', this, () => {
+      const score = this.score;
+      const competences = this.get('details.competences');
+      const newScore = competences.reduce((value, competence) => {
+        value += (typeof competence.juryScore !== 'undefined' && competence.juryScore !== false) ? competence.juryScore : competence.obtainedScore;
+        return value;
+      }, 0);
+      if (newScore !== score) {
+        this.set('juryScore', newScore);
+      } else {
+        this.set('juryScore', false);
+      }
+    });
```

Comment:
Difficile d'octaniser ici à cause du "schedule" , j'ai l'impression que ça met le bazarre dans le contexte du "this" ...